docs/source/users/index.md

```diff
@@ -342,14 +344,29 @@ providers, as defined in [`langchain.llms`](https://langchain.readthedocs.io/en/
 - `openai-chat`
 - `sagemaker-endpoint`
 
-You can find a list of supported providers and models by running `%ai list`. Some providers
-define a list of supported models. If a provider does not define a list of supported models,
-consult the vendor's documentation. The [Hugging Face web site](https://huggingface.co/)
-includes a list of models, for example.
+:::{warning}
+As of v0.8.0, only the `%%ai` *cell* magic may be used to invoke a language
+model, while the `%ai` *line* magic is reserved for invoking subcommands.
+:::
+
+### Listing available models
+
+Jupyter AI also includes multiple subcommands, which may be invoked via the
+`%ai` *line* magic. Jupyter AI uses subcommands to provide additional utilities
+in notebooks while keeping the same concise syntax for invoking a language model.
```

Comment:
Probably OK to delete this, or to create a new small subsection about subcommands. In this new subsection, mention `%ai help`. (As of this PR, `%ai help`'s output will be small, but #136 and #170 will add to it.)

Reply:
I anticipate this being a major point of confusion for users, so I think it's important we note this at the outset of this section where we introduce line magics as opposed to cell magics. As you've pointed out, we're adding new subcommands very shortly, so it's still plenty relevant enough to remain where it is.